backend/core/taps/bgpstreamhist.py

```diff
@@ -82,6 +83,7 @@ def parse_bgpstreamhist_csvs(prefixes_file=None, input_dir=None):
                                                     routing_key="update",
                                                     serializer="json",
                                                 )
+                                                time.sleep(0.1)
```

Comment:
why was this necessary?

Reply:
I just put it so that we do not just flood the detector(s) with 1000s or 100s of 1000s of pre-recorded messages at once.

Reply:
Do you think it is OK or does it hurt? I thought that since we replayed history proper operation is more imporant than speed (I suppose that without this mini rate-limiting ARTEMIS detectors and RMQ can be easily flooded)